lib/virtfs/ext3/file.rb

```diff
@@ -0,0 +1,26 @@
+module VirtFS::Ext3
+  class File
+    def initialize(dir_entry, superblock)
+      @bs = superblock
+      @de = dir_entry
+    end
+
+    def to_h
+      { :directory? => @de.dir?,
+        :file?      => @de.file?,
+        :symlink?   => @de.symlink? }
+    end
+
+    def fs
+      @de.fs
+    end
+
+    def size
+      @de.size
+    end
+
+    def close
+      @de.close
+    end
+  end # class File
```

Comment:
@roliveri I have a test case reproducing the issue due to the missing 'raw_read' method and am working on a fix. To do so, I'll have to change the ext3 module's File and file_class_methods to behave similarily to nativefs where 'lookup_file' returns an instance of the 'file_obj' which 'sysread' can be called on (or the equivalent mechanism for the ext3 filesystem).

The issue w/ this is that it's in conflict w/ our original goal of having the first pass of this module be a direct mapping of manageiq/vmdb functionality. As it stands the File class instance in created through file_class_methods using the get_file helper which is an exact line-by-line port of the manageiq getFile method (taking into account style fixes and such).

We can roll this enhancement into this PR w/ the others, but I feel it'd be best to go w/ our original strategy, and do the port first, then enhancement the new virtfs-ext3 module w/ new funcitonaility bit by bit. Thought?